onnxscript/ir/_core.py

```diff
@@ -1845,6 +1849,8 @@ def insert_after(self, node: Node, new_nodes: Iterable[Node] | Node, /) -> None:
     def insert_before(self, node: Node, new_nodes: Iterable[Node] | Node, /) -> None:
         """Insert new nodes before the given node in O(#new_nodes) time.
 
+        A unique name will be assigned to the node and its values is not yet specified.
```

Comment:
Not quite sure what the (is not yet specified) part means.

Reply:
Thanks, updated to "Unique names will be assigned to the node and its values if any name is ``None``." Does that look better?